### PR TITLE
Fix sdcsrm 626 add google analytics

### PR DESCRIPTION
**Pull request by kacperpONS**
# Motivation and Context

All the same as this PR: https://github.com/ONSdigital/srm-rh-ui/pull/52 but removing an old comment in the js code, and updating the used cookies table for GA based on their docs: https://support.google.com/analytics/answer/11397207?hl=en.
I've also made some small changes to the translations part of the README.

Has to be on another branch because the old one has been squashed and merged.

# What has changed

# How to test?

# Links

https://jira.ons.gov.uk/browse/SDCSRM-626

# Screenshots (if appropriate):

### CODE REVIEW

File: rh_ui/templates/info_pages/cookies.html [hunk 1]
```diff
@@ -106,26 +106,13 @@ <h3>{{ _("Cookies that measure website use") }}</h3>
                         {
                             "tds": [
                                 {
-                                    "value": "<code>_gid</code>"
+                                    "value": "<code>_ga_<container-id></code>"
```
Comment (ryangrundy7) [inline rh_ui/templates/info_pages/cookies.html:109]: This isn't been showing correctly in the page

File: rh_ui/translations/cy/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -716,3 +706,18 @@ msgstr "Mae rhywun eisoes wedi cyflwyno ymateb gan ddefnyddio’r cod mynediad h
...
+
+#~ msgid "24 hours"
+#~ msgstr ""
+
+#~ msgid "Used to manage the rate at which page view requests are made"
+#~ msgstr ""
+
+#~ msgid "10 minutes"
+#~ msgstr ""
+
```
Comment (ryangrundy7) [inline rh_ui/translations/cy/LC_MESSAGES/messages.po:723]: Can this been removed?

File: rh_ui/templates/info_pages/cookies.html (hunk 1 above)
Comment (AdamHawtin) [inline rh_ui/templates/info_pages/cookies.html:109]: When viewing the page this is only rendering as `_ga_`, I think the angle braces probably need to be escaped.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: rh_ui/translations/cy/LC_MESSAGES/messages.po (hunk 2 above)
Comment (kacperpONS) [reply rh_ui/translations/cy/LC_MESSAGES/messages.po]: It's the translation of the description of the old cookies that we don't need anymore. Removed it 👍

File: README.md [hunk 3]
```diff
@@ -28,37 +28,44 @@ or to run it in Pycharm, use the run template that's specified and it should wor
...
 required language code. Only generate a individual language file once.
 
-Note that this implementation includes an English translation. This is needed due to an issue with implementing pybabel
-with aiohttp.
-
 ```
 pipenv run pybabel init -i rh_ui/translations/messages.pot -d rh_ui/translations -l cy
 ```
 
+3. Update tranlsation files
```
Comment (ryangrundy7) [inline README.md:54]:
```suggestion
3. Update translation files
```

File: README.md [hunk 4]
```diff
@@ -28,37 +28,44 @@ or to run it in Pycharm, use the run template that's specified and it should wor
...
+
 Once created, you can update the existing language messages.po files to include changes in the messages.pot by running
 the following. This will update ALL language files.
 
 ```
 pipenv run pybabel update -i rh_ui/translations/messages.pot -d rh_ui/translations
 ```
 
+4. Complile the translations
```
Comment (ryangrundy7) [inline README.md:63]:
```suggestion
4. Compile the translations
```

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**